### ISSUE TITLE
Travis CI: go mod issue

### ISSUE THREAD

**Issue by jtopjian** [gophercloud/gophercloud#1728]
It looks like there's a dependency issue with the go modules:

```
golang.org/x/tools/go/gcexportdata
../../../golang.org/x/tools/go/gcexportdata/gcexportdata.go:103:12: undefined: gcimporter.IExportData
```

**Comment by jtopjian** [comment]
This looks to be an issue specific to Go 1.10. We might have to drop Go 1.10 from Travis, which should be ok.

**Comment by jtopjian** [comment]
Fixed in #1729
